test/statsd_test.rb

```diff
@@ -65,7 +65,7 @@ def test_statsd_measure_returns_return_value_of_block
     assert_equal 'sarah', return_value
   end
 
-  def test_statsd_measure_returns_return_value_of_block_even_if_nil
```

Comment:
This method is defined elsewhere already, with a slightly different implementation.